tests/system/Encryption/Handlers/OpenSSLHandlerTest.php

```diff
@@ -20,10 +21,7 @@
  */
 final class OpenSSLHandlerTest extends CIUnitTestCase
 {
-    /**
-     * @var \CodeIgniter\Encryption\Encryption
-     */
-    protected $encryption;
+    protected \CodeIgniter\Encryption\Encryption $encryption;
```

Comment:
Why is this full classname?

Reply:
that's possibly a rector bug due to found `use Config\Encryption as EncryptionConfig` exists that marked as conflict to the class name, instead of to the alias.

Reply:
fixed https://github.com/codeigniter4/CodeIgniter4/pull/5813/commits/977defed45be74def619915bb5390fee789056e2